File: AspNetMvcUnleashed/0672329980_source/Chapter12/CS/MvcApplication1/MvcApplication1/Controllers/LookupController.cs
using System.Web.Mvc;
using System.Web.Security;

namespace MvcApplication1.Controllers
{
    public class LookupController : Controller
    {

        public ActionResult Index(string search)
        {
            MembershipUserCollection users = new MembershipUserCollection();
            if (!string.IsNullOrEmpty(search))
                users = Membership.FindUsersByName("%" + search + "%");
            
            return View(users);
        }

    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter12/VB/MvcApplication1/MvcApplication1/Controllers/LookupController.vb
Public Class LookupController
    Inherits Controller

    Public Function Index(ByVal search As String) As ActionResult
        Dim users As New MembershipUserCollection()
        If Not String.IsNullOrEmpty(search) Then
            users = Membership.FindUsersByName("%" & search & "%")
        End If

        Return View(users)
    End Function

End Class
